src/main/java/carpet/utils/IEntity.java

```diff
@@ -0,0 +1,6 @@
+package carpet.utils;
```

Comment:
Could you please put this one as carpet.fakes.EntityInterface. I am trying to avoid mixing useless interfaces for the sake of mixins and useful utils.